.changeset/brown-eggs-remember.md

```diff
@@ -0,0 +1,33 @@
+---
+"@comet/admin": minor
+---
+
+Add `gridColumnTypes`, a group of helpers for using the "date" and "dateTime" types in Data Grid
```

Comment:
I wouldn't restrict this to date and date time. Maybe we'll need this for something else in the future.

Reply:
I changed it to the following now:

> Add the `dataGridDateColumn` and `dataGridDateTimeColumn` helpers for using the "date" and "dateTime" types in Data Grid

I think this should be fine, as it doesn't mean other helpers will or will not be added in the future.